client/src/components/Video/index.js

```diff
@@ -0,0 +1,65 @@
+import {Button, Typography} from 'antd';
+import React, {useEffect} from "react";
+import {API_URL} from "../../App";
+import styles from "./index.module.css";
+
+const {Text} = Typography;
+const Video = (props) => {
+    const {video: {id, title, url, rating: videoRating, timestamp}, handleDeleteVideo} = props;
+    const [rating, setRating] = React.useState(videoRating);
+
+    const videoId = url.split("v=")[1].substring(0, 11);
```

Comment:
What's special about the first 11 characters? It may be worth including a comment describing why you're stripping off the end, and why that's the right place to stop.